Add fr-Fr locale for landing page

oleole39 opened the pull request:
Hello,
Here is the French translation for the landing page. 
Note that I have slightly modified the title in literally : "Transcribe. Automatically and locally".

thewh1teagle commented:
Looks good overall!
How about using `Transcription Locale Audio` in the title? since currently it looks too long in the website

oleole39 commented:
> How about using `Transcription Locale Audio` in the title? since currently it looks too long in the website
It can work, better though with this words' order: `Transcription audio locale`.
I updated the PR. Also added the word "offline" in the description to emphasize.

oleole39 commented:
One other detail for consistency: the landing page's description field talks about video file. However, in the UI the button used to prompt for an input file is called "Select an audio file". However it seems to work with video files too (It worked for me with `webm`, `mp4`, `mkv` containers successfully, but failed with `wmv`) ?

thewh1teagle commented:
> One other detail for consistency: the landing page's description field talks about video file. However, in the UI the button used to prompt for an input file is labelled "Select an audio file". However it seems to work with video files too (It worked for me with `webm`, `mp4`, `mkv` and `avi` containers successfully, but failed with `wmv`) ?

I chose to use `Select audio file` to make it clear that the feature is for transcribing audio. I thought this will help avoid any confusion with non-audio or non-video files. Do you have a better suggestion, maybe `Select files`?

As for the formats Vibe supports, it can handle almost any audio or video files that `ffmpeg` supports.

oleole39 commented:
> Do you have a better suggestion, maybe `Select files`?

Yes I think `Select file`, or maybe `Select media file` if you'd prefer, would be less restrictive. In a similar way, note that in Linux Mint the file explorer's window that pops up after clicking the button hints at "Audio" in the section which shows allowed extensions, although that filter seems to include video file extensions as well. 

> Btw, I updated the landing website, also now it's more easier to use the `i18 ally` extension with landing translation, just need to open `VSCode` in `landing` folder directly

Thanks for the notice. I must confess that so far I have used some dumb text editor (my pull requests so far have been about small changes).

thewh1teagle commented:
> note that in Linux Mint the file explorer's window that pops up after clicking the button hints at "Audio"

Good catch! I didn't notice it, I believe it's been there since version 0.0.1

Fixed in the latest commits.

thewh1teagle commented:
Merged & released
Thanks for your contribution!

oleole39 commented:
I see some new translations strings got added since the PR (the translation would need to be corrected for some of them), but as I notice you are in the process of developing the landing page, I will wait for all the strings to be available to adapt the fr-FR locale.
